<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassWidget</class>
 <widget class="QWidget" name="ClassWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lbl_name">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lw_attrs"/>
   </item>
   <item>
    <widget class="QListWidget" name="lw_meths"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
